resteasy-wadl/src/test/java/org/jboss/resteasy/test/nextgen/wadl/WadlTests.java

```diff
@@ -44,24 +43,25 @@ public void setClient(Client client) {
       this.client = client;
    }
 
-   public String getUrl() {
-      return url;
-   }
+   private static ResteasyWadlDefaultResource defaultResource = new ResteasyWadlDefaultResource();
 
-   public void setUrl(String url) {
-      this.url = url;
-   }
 
    @BeforeClass
    public static void before() throws Exception {
 
       httpServer = HttpServer.create(new InetSocketAddress(TestPortProvider.getPort()), 10);
       contextBuilder = new HttpContextBuilder();
       contextBuilder.getDeployment().getActualResourceClasses().add(BasicResource.class);
-      contextBuilder.getDeployment().getActualResourceClasses().add(ResteasyWadlDefaultResource.class);
+//      contextBuilder.getDeployment().getActualResourceClasses().add(ResteasyWadlDefaultResource.class);
+//      contextBuilder.getDeployment().getActualResourceClasses().add(ResteasyWadlExtendedResource.class);
       contextBuilder.getDeployment().getActualResourceClasses().add(RESTEASY1246.class);
+      contextBuilder.getDeployment().getActualResourceClasses().add(ExtendedResource.class);
+
       contextBuilder.bind(httpServer);
-      ResteasyWadlDefaultResource.getServices().put("/", ResteasyWadlGenerator.generateServiceRegistry(contextBuilder.getDeployment()));
+//      defaultResource.getServices().put("/", ResteasyWadlGenerator.generateServiceRegistry(contextBuilder.getDeployment()));
```

Comment:
Can we also remove the commented out lines here above or do they have some special meaning that justify their presence?